sha1/src/checked/ubc_check.rs

```diff
@@ -0,0 +1,1179 @@
+//! Direct translation of the C code
+//! Original license and comments:
+//!
+//! Copyright 2017 Marc Stevens <marc@marc-stevens.nl>, Dan Shumow <danshu@microsoft.com>
+//! Distributed under the MIT Software License.
+//! See accompanying file LICENSE.txt or copy at
+//! https://opensource.org/licenses/MIT
+//!
+//! this file was generated by the 'parse_bitrel' program in the tools section
+//! using the data files from directory 'tools/data/3565'
+//!
+//! sha1_dvs contains a list of SHA-1 Disturbance Vectors (DV) to check
+//! dvType, dvK and dvB define the DV: I(K,B) or II(K,B) (see the paper)
+//! dm[80] is the expanded message block XOR-difference defined by the DV
+//! testt is the step to do the recompression from for collision detection
+//! maski and maskb define the bit to check for each DV in the dvmask returned by ubc_check
```

Comment:
I think it would be helpful if this were moved to be a doc comment on `SHA1_DVS`

Reply:
Also is `testt` a typo?

Reply:
> Also is `testt` a typo?

I don't think so, it is the name used everywhere in the original code

Reply:
Maybe put backticks around it so it looks intentional?

Reply:
fixed